src/ActionModal/components/ActionModal.vue

```diff
@@ -830,6 +844,20 @@ export default {
     maxDecimals(value, decimals) {
       return Number(BigNumber(value).toFixed(decimals)) // TODO only use bignumber
     },
+    alternativeFeeSelector() {
+      const alternativeFeeBalance = this.balances.find(
+        ({ denom }) =>
+          denom !== this.getDenom && this.amount >= this.estimatedFee
```

Comment:
the fee is different in every denom I gues

Reply:
oh good point!